easybuild/easyconfigs/c/CIRCexplorer2/CIRCexplorer2-2.3.3-intel-2018a-Python-2.7.14.eb

```diff
@@ -0,0 +1,38 @@
+easyblock = 'PythonPackage'
+
+name = 'CIRCexplorer2'
+version = '2.3.3'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://circexplorer2.readthedocs.io/'
+description = "CIRCexplorer2 is a comprehensive and integrative circular RNA analysis toolset."
+
+toolchain = {'name': 'intel', 'version': '2018a'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+checksums = ['30f9273e4a9ab1575a001d1165ba19972995f68d2acbfbc1b113fca49054d8ae']
+
+dependencies = [
+    ('Python', '2.7.14'),
+    ('TopHat', '2.1.1', versionsuffix),
```

Comment:
I got word that using `TopHat` 2.1.1 leads to problems with `tophat-reports`.

According to https://www.biostars.org/p/186979/, the recommended workaround is to use `TopHat` 2.1.0 instead...